### PR TITLE
WT-7828 Run many-coll-test on different variant

### PR DESCRIPTION
**Pull request by etienneptl**
Moved many-collection-tests to run on `ubuntu1804-wt-large` to be able to set `ulimit -n` to a greater value than the default one.

### CODE REVIEW

File: test/evergreen.yml [hunk 1]
```diff
@@ -2537,6 +2537,8 @@ tasks:
           script: |
             set -o errexit
             set -o verbose
+            sudo su
+            ulimit -n 1000000
```
Comment (lukech) [inline test/evergreen.yml:2541]: I believe it's an anti-pattern to update system-level settings in an Evergreen task. Did you check with the Build team to see whether they could write the required ulimit value into the distro image? Our team should be the only user for the `ubuntu1804-wt-large` distro.

Comment (etienneptl) [reply test/evergreen.yml:2541]: I asked them the question if it was possible, will keep you updated.

Comment (etienneptl) [reply test/evergreen.yml:2541]: See https://jira.mongodb.org/browse/BUILD-13585

**Review by lukech — APPROVED** [review]

File: test/evergreen.yml (hunk 1 above)
Comment (etienneptl) [reply test/evergreen.yml:2543]: As discussed, we will leave this for now and remove it as part of https://jira.mongodb.org/browse/WT-7842

File: test/evergreen.yml [hunk 2]
```diff
@@ -2744,6 +2748,20 @@ buildvariants:
...
+  display_name: "Many collection tests"
+  batchtime: 480 # 3 times a day
+  run_on:
+  - ubuntu1804-wt-large
+  expansions:
+    make_command: PATH=/opt/mongodbtoolchain/v3/bin:$PATH make
+    posix_configure_flags:
+      --enable-silent-rules --enable-python --enable-zlib --enable-snappy
+      --enable-strict --enable-static --prefix=$(pwd)/LOCAL_INSTALL
+    test_env_vars:
```
Comment (lukech) [inline test/evergreen.yml:2762]: @etienneptl Just realized the whole purpose of this new build variant is to run on a different distro. Actually we can reuse the existing `large-scale-tests` build variant and overload the distro for a given task using `distro:` argument setting.

Comment (etienneptl) [reply test/evergreen.yml]: Thanks, looks way better, see c17574a

**Review by sulabhM — APPROVED** [review]

**Pull request merged**